flytekit/bin/entrypoint.py

```diff
@@ -17,6 +19,16 @@
 from flytekit.models import literals as _literal_models
 
 
+def _run_with_retries(func, expected_exception, max_retries=10):
```

Comment:
Do we not have something that does this already :-D ? I think there is a whole python library to do retries.. not sure if we use it...

Reply:
Changing to using the retry library